### DISCUSSION TITLE
关键信息提取中，预训练模型有linking属性，inference模型没有linking属性，关联不上key和value

### DISCUSSION THREAD

**Discussion by achial** [PaddlePaddle/PaddleOCR#12241] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：mac
- 版本号/Version：Paddle：2.5.1 PaddleOCR：2.7.0.2  问题相关组件/Related components：Name: paddlehub Version: 2.3.0
- 运行指令/Command Code： hub serving start -m kie_ser_re -p 8866
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

在文档中
https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/ppstructure/kie/README_ch.md
只有执行这个命令才有linking属性来表示key和value 的关系
我不懂python，经社群朋友的引导，简单看了下源码，好像只有预训练的ser才有linking属性，我怎么才能让paddlehub启动的kie_ser_re模块，输入图片之后返回的json带着key和value关系呢

  python3 ./tools/infer_kie_token_ser_re.py \
  -c configs/kie/vi_layoutxlm/re_vi_layoutxlm_xfund_zh.yml \
  -o Architecture.Backbone.checkpoints=./pretrained_model/re_vi_layoutxlm_xfund_pretrained/best_accuracy \
  Global.infer_img=./train_data/XFUND/zh_val/val.json \
  Global.infer_mode=False \
  Global.use_gpu=False \
  -c_ser configs/kie/vi_layoutxlm/ser_vi_layoutxlm_xfund_zh.yml \
  -o_ser Architecture.Backbone.checkpoints=./pretrained_model/ser_vi_layoutxlm_xfund_pretrained/best_accuracy

 {"transcription": "姓名:", "label": "question", "points": [[259, 541], [375, 541], [375, 590], [259, 590]], "id": 30, "linking": [[30, 26]], "bbox": [259, 541, 375, 590], "pred_id": 1, "pred": "QUESTION"}, {"transcription": "夏艳辰", "label": "answer", "points": [[410, 541], [535, 541], [535, 590], [410, 590]], "id": 26, "linking": [[30, 26]], "bbox": [410, 541, 535, 590], "pred_id": 3, "pred": "ANSWER"}, {"transcription": "出生日期:", "label": "question", "points": [[260, 626], [481, 626], [481, 684], [260, 684]], "id": 34, "linking": [[34, 635]], "bbox": [260, 626, 481, 684], "pred_id": 1, "pred": "QUESTION"}, {"transcription": "1983/6/8", "label": "answer", "points": [[494, 630], [841, 630], [841, 675], [494, 675]], "id": 635, "linking": [[34, 635]], "bbox": [494, 630, 841, 675], "pred_id": 3, "pred": "ANSWER"}

**Comment by cwmore** [comment]
见这个： https://github.com/PaddlePaddle/PaddleOCR/discussions/13840
